src/test/groovy/stix/datamarkings/TLPmarkingsSpec.groovy

```diff
@@ -0,0 +1,96 @@
+/**
+ * 
+ */
+package stix.datamarkings
+
+import org.skyscreamer.jsonassert.JSONAssert
+import org.skyscreamer.jsonassert.JSONCompareMode
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+
+import io.digitalstate.stix.common.StixInstant
+import io.digitalstate.stix.datamarkings.MarkingDefinition
+import io.digitalstate.stix.datamarkings.objects.Tlps
+import io.digitalstate.stix.json.StixParsers
+import io.digitalstate.stix.sdo.objects.Indicator
+import spock.lang.Shared
+import spock.lang.Specification
+
+/**
+ * @author Kathy Lee Simunich, Argonne National Laboratory
+ *
+ */
+class TLPmarkingsSpec extends Specification {
+	
+	@Shared ObjectMapper mapper = new ObjectMapper()
+	
+	
+	def "test TLP"() {
+		when: "create tlp white"
+		
+		MarkingDefinition originalMarkingDefinition = Tlps.TLP_WHITE;
+		
+        then: "Convert Marking Definition to Json"
+            JsonNode originalJson = mapper.readTree(originalMarkingDefinition.toJsonString())
+            String originalJsonString = mapper.writeValueAsString(originalJson)
+            println "Original Json: ${originalJsonString}"
+
+        then: "Parse Json back into Marking Definition Object"
+            MarkingDefinition parsedMarkingDefinition = (MarkingDefinition)StixParsers.parseObject(originalJsonString)
```

Comment:
- [ ] Update mock Marking Def to remove the custom props on the TLPs as per @anlklsim comments

Reply:
- [ ] Add usage of the Generics parser StixParser.parse() for testing